### PR TITLE
Setup CI for Serverless release 1.18

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title. This includes builds on 4.6 through 4.9.

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.8.yaml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: operator-e2e-aws-ocp-48
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.8.yaml:97]: You don't want cluster pools for 4.8 ?

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.9.yaml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: operator-e2e-aws-ocp-49
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.9.yaml:99]: No cluster pools for 4.9 either?

**Review by mgencur — COMMENTED** [review]
Just two comments about cluster pools. Otherwise looks good.

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.8.yaml (hunk 1 above)
Comment (mgencur) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.8.yaml:97]: Alright. It's not necessary. I'm just asking if this was intentional or you'd want cluster pools. I can mark this as lgtm

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.9.yaml (hunk 2 above)
Comment (markusthoemmes) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.9.yaml:99]: I'd rather not use a release a test-ballon for those tbh. Are we running any other 4.9 jobs on pools already?

**Pull request merged**